nms/app/packages/magmalte/grafana/dashboards/XWFMDashboards.js

```diff
@@ -19,8 +19,9 @@ import type {GrafanaDBData} from './Dashboards';
 
 export const XWFMDBData = (networks: Array<string>): GrafanaDBData => {
   return {
-    title: 'XWF-M Container/Node Stats',
-    description: '',
+    title: 'Container/Node Stats',
+    description:
+      'This dashboard shows stats from node_exporter and cAdvisor if those services are installed on your gateways.',
```

Comment:
minor nit: This dashboard shows stats from node_exporter and cAdvisor if those services are installed on your gateways -> This dashboard shows stats from node_exporter and cAdvisor.  

Do we need additional comment regarding installation on the dashboard page?

Reply:
Yeah, since this is going to be provided to all CWF networks, they may not have these services installed. This will make it more clear why that's the case.